dashboard/config/levels/custom/maze/2-3 Bee Functions 2.level

```diff
@@ -1,5 +1,6 @@
 <Karel>
   <config><![CDATA[{
+  "published": true,
```

Comment:
I used the levelbuilder interface to edit the levels, but I manually reverted `created_at` on line 5 so that we don't lose that meta-data.